jest.config.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
   },
   moduleNameMapper: {
     '.+\\.(css|styl|less|sass|scss)$': 'identity-obj-proxy',
-    '.+\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$': '<rootDir>/__mocks__/file-mock.js',
+    '.+\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$': '<rootDir>/fileTransformer.js',
```

Comment:
using _mocks_/file-mock.js doesn't seem to work here (or potentially exist?) with the svg file of the edx logo. It thew a configuration error when running tests in programListPage. I added the fileTransformer file which seems to work a little better. I made this change for testing image files based on jest documentation that I found worked. 

https://jestjs.io/docs/en/webpack